src/ai/backend/manager/raft/utils.py

```diff
@@ -0,0 +1,108 @@
+import pickle
+from typing import Any
+
+from aiohttp import web
+from aiohttp.web import RouteTableDef
+from raftify import (
+    Raft,
+    set_confchange_context_deserializer,
+    set_confchangev2_context_deserializer,
+    set_entry_context_deserializer,
+    set_entry_data_deserializer,
+    set_fsm_deserializer,
+    set_log_entry_deserializer,
+    set_message_context_deserializer,
+    set_snapshot_data_deserializer,
```

Comment:
I agree with your point. It would be more helpful for code readability to register callbacks all at once, rather than the current method.

I will make improvements in the PR below.
https://github.com/lablup/raftify/pull/101